addons/im_livechat/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -17,7 +17,10 @@ registerInstancePatchEntity('MessagingNotificationHandler', 'im_livechat.messagi
      */
     _handleNotificationChannelTypingStatus(data) {
         const { channelId, partner_id } = data;
-        const channel = this.env.entities.Thread.insert({ id: channelId, model: 'mail.channel' });
+        const channel = this.env.models['mail.thread'].insert({
+            id: channelId,
+            model: 'mail.channel',
```

Comment:
We're not talking about the same model here, this is a bit strange.

Reply:
Maybe all fields referencing python models should be res_model

Reply:
Or py_model...

Reply:
Or server_model since that's the prefix we started using for other stuff related to server